frontend/src/metabase/parameters/utils/dashboards.js

```diff
@@ -204,3 +204,29 @@ export function hasMatchingParameters({
     return dashcardMappingsByParameterId[parameter.id] != null;
   });
 }
+
+export function getFilteringParameterValuesMap(parameter, parameters) {
+  const { filteringParameters = [] } = parameter || {};
+  const filteringParameterValues = Object.fromEntries(
+    parameters
+      .filter(p => filteringParameters.includes(p.id) && p.value != null)
+      .map(p => [p.id, p.value]),
+  );
+
+  return filteringParameterValues;
+}
+
+export function getParameterValuesSearchKey({
+  dashboardId,
+  parameterId,
+  query = null,
+  filteringParameterValues = {},
+}) {
+  const sortedParameterValues = _.sortBy(
+    Object.entries(filteringParameterValues),
+    "0",
+  );
```

Comment:
Not sure I follow what does sorting by `"0"` means here. Does it sort an entries list by a parameter ID (first entry element)? I'm thinking if it'd make it easier to read if we define a constant / add a comment explaining what's happening. Something like:

```js
const BY_PARAMETER_ID = "0"
const sortedParameterValues = _.sortBy(
  Object.entries(filteringParameterValues),
  BY_PARAMETER_ID,
);
```

Reply:
Good idea -- added that const + a comment explaining the sort